src/main/java/dansplugins/factionsystem/services/MessageService.java

```diff
@@ -0,0 +1,44 @@
+package dansplugins.factionsystem.services;
+
+import dansplugins.factionsystem.MedievalFactions;
+import org.bukkit.configuration.InvalidConfigurationException;
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.configuration.file.YamlConfiguration;
+
+import java.io.File;
+import java.io.IOException;
+
+public class MessageService {
+
+
+    private File languageFile;
+    private FileConfiguration language;
+
+    public void createLanguageFile() {
+        languageFile = new File(new MedievalFactions().getMedievalFactions().getDataFolder(), "language.yml");
```

Comment:
Pass the plugin instance into the message service.

Reply:
Yeah pass the main class to MessageService.java rather than instantiating a new instance here.